selfdrive/car/honda/carcontroller.py

```diff
@@ -137,6 +144,12 @@ def update(self, enabled, CS, frame, actuators,
     # Send CAN commands.
     can_sends = []
 
+    # tester present - w/ no response (keeps radar disabled)
+    if CS.CP.carFingerprint in HONDA_BOSCH and CS.CP.openpilotLongitudinalControl:
+      if (frame % 10) == 0:
+        can_sends.append((0x18DAB0F1, 0, b"\x02\x3E\x80\x00\x00\x00\x00\x00", 1))
+        # TODO: Add filtering in panda to only allow this specific message from the hondaBosch safety mode
```

Comment:
Also add radar message to the harness malfunction check in panda to prevent staying engaged when the radar comes back.